go.sum

```diff
@@ -115,10 +119,14 @@ github.com/lightninglabs/neutrino v0.0.0-20190313035638-e1ad4c33fb18/go.mod h1:v
 github.com/lightninglabs/neutrino v0.0.0-20190313220016-a6cc2a8361d5/go.mod h1:NXpKUhcszEdn7v7ze5vsnSHI1KX6HqFdNRu0eqhkJzk=
 github.com/lightninglabs/neutrino v0.0.0-20190314214430-643615b8c132 h1:yaKXc3D/qeEy2wv+bIQwmrq+kThdbWg0p2ggea3RAsg=
 github.com/lightninglabs/neutrino v0.0.0-20190314214430-643615b8c132/go.mod h1:NXpKUhcszEdn7v7ze5vsnSHI1KX6HqFdNRu0eqhkJzk=
+github.com/lightninglabs/neutrino v0.0.0-20190321023416-6dac90b98052 h1:YVi5x7xCumfIVtEejy3zBAej+NqY9EONy/fRim2+azQ=
```

Comment:
I think if you run `go mod tidy`, then these duplicates will go away.

Reply:
Ran `go mod tidy` and this is the delta that remains.